source/type/pb_converter.cpp

```diff
@@ -49,6 +49,538 @@ namespace nt = navitia::type;
 namespace pt = boost::posix_time;
 namespace gd = boost::gregorian;
 
+//*********************************************
+namespace ProtoCreator{
+
+template<typename NT, typename PB>
+static void fill_codes(const NT* nt, const nt::Data& data, PB* pb) {
+    for (const auto& code: data.pt_data->codes.get_codes(nt)) {
+        for (const auto& value: code.second) {
+            auto* pb_code = pb->add_codes();
+            pb_code->set_type(code.first);
+            pb_code->set_value(value);
+        }
+    }
+}
+
+static void fill_property(const std::string& name,
+                          const std::string& value,
+                          pbnavitia::Property* property) {
+    property->set_name(name);
+    property->set_value(value);
+}
+
+PbCreator::Filler PbCreator::Filler::copy_depth_decr(){
+    if (depth == 0) {
+        return PbCreator::Filler(0,dump_message, pb_creator);
+    }
+    return PbCreator::Filler(depth-1,dump_message, pb_creator);
+}
+
+void PbCreator::Filler::fill_pb_object(const navitia::type::Contributor* cb, pbnavitia::Contributor* contrib){
+    if(cb == nullptr)
+        return;
```

Comment:
please always use {}. You can put everything on the same line for short things like this:
```c++
    if (cb == nullptr) { return; }
```